.github/workflows/diff.yml

```diff
@@ -0,0 +1,36 @@
+name: git diff with nbdime
+
+on:
+  push:
+    branches:
+      - main
+      - staging
+  pull_request:
+    branches:
+      - main
+      - staging
+
+permissions:
+  contents: read
+
+jobs:
+  diffs:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+    - name: Set up Python 3.9
+      uses: actions/setup-python@v3
+      with:
+        python-version: "3.9.13"
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install nbdime ipython_genutils
+    - name: Configure nbdime
+      run: |
+        /opt/hostedtoolcache/Python/3.9.13/x64/bin/nbdime config-git --enable --global
+    - name: git diff
+      run: |
+        git diff HEAD^ HEAD
```

Comment:
Quick question: do you know if this always shows the diff for (just) the latest commit?

I think it'd be more useful to have the diff with the target branch.

If it's easier, you can ignore the `staging` branch and just use this with `main` since that's the only one really used today.

Reply:
I'll take a look

Reply:
I believe It works as expected albeit a bit counter intuitive. Played around with it here https://github.com/raybellwaves/nbdime_gha_test/pull/12 and was able to replicate the GHA locally.

The actions/checkout@v3 checkous out the PR (as a new commit) and HEAD^ is set as the main branch

`HEAD is now at 60db89e Merge 7ad25b9d151fb152a716f409409d93901ae35574 into dbe429390474c36a2ef8aaa7f30125ba2d88a143`

Reply:
Cool, thanks!